extensions/exchanges/bittrex/exchange.js

```diff
@@ -1,9 +1,8 @@
-var bittrex_authed = require('node.bittrex.api'),
-    bittrex_public = require('node.bittrex.api'),
-    path = require('path'),
-    moment = require('moment'),
-    n = require('numbro'),
-    colors = require('colors')
+var bittrex_authed = require('node-bittrex-api'),
+  bittrex_public = require('node-bittrex-api'),
+  moment = require('moment'),
+  n = require('numbro')
```

Comment:
nit: indenting?

Reply:
It is formatted according the the setting in .eslintrc.
I'll manually format it.